reconcile/utils/terrascript_client.py

```diff
@@ -3155,6 +3166,145 @@ def is_elasticsearch_domain_name_valid(name):
         pattern = r'^[a-z][a-z0-9-]+$'
         return re.search(pattern, name)
 
+    @staticmethod
+    def elasticsearch_log_group_identifier(
+        domain_identifier: str,
+        log_type: ElasticSearchLogGroupType
+    ) -> str:
+        log_type_name = log_type.value.lower()
+        return f'OpenSearchService__{domain_identifier}__{log_type_name}'
+
+    def _elasticsearch_aggregate_log_groups_per_account(
+        self
+    ) -> Iterable[Tuple[str, str, str, str]]:
+        log_group_identifiers = []
+        for resources in self.account_resources.values():
+            for i in resources:
+                res = i.get('resource')
+                ns = i.get('namespace_info')
+                if not ns.get('managedTerraformResources'):
+                    continue
+                if res.get('provider') != 'elasticsearch':
+                    continue
+                # res.get('', []) won't work, as publish_log_types is
+                # explicitly set to None if not set
+                log_types = res['publish_log_types']
+                if not log_types:
+                    log_types = []
+                for log_type in log_types:
+                    region = ns.get('cluster').get('spec').get('region')
```

Comment:
What is the source for `ns`? Is it a graphql query? If so, you'll be getting weird exceptions and `get` will only be more surprising, see https://issues.redhat.com/browse/APPSRE-3868 . I'd suggest you go with normal dictionary accesses:
```python
ns['cluster']['spec']['region'`]
```

Reply:
Yeah good point. Already stumbled across that issue during development.

Converted to standard dict access.